### PR TITLE
Initial cut at GHA sciplat-lab build

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

File: pipelines/sqre/infra/build_sciplatlab.groovy [hunk 1]
```diff
@@ -16,91 +16,57 @@ node('jenkins-manager') {
...
+      ]
+      def json = new groovy.json.JsonBuilder(body)
 
-    // ensure the current image is used
-    stage('docker pull') {
-      docImage = "${baseImage}:${tagPrefix}${tag}"
-      docker.image(docImage).pull()
-    }
+      def url = new URL("https://api.github.com/repos/lsst-sqre/sciplat-lab/actions/workflows/build.yaml/dispatches")
+      println("url: ${call}")
```
Comment (ktlim) [inline pipelines/sqre/infra/build_sciplatlab.groovy:50]: You didn't catch my error here 😄

File: pipelines/sqre/infra/build_sciplatlab.groovy [hunk 2]
```diff
@@ -16,91 +16,57 @@ node('jenkins-manager') {
...
-      ])
-    }
+    stage('trigger GHA') {
+      def body = [
+        ref: branch,
+        inputs: [
+          tag: tag,
+          supplementary: supplementary,
+          image: image,
+	  push: push
```
Comment (ktlim) [inline pipelines/sqre/infra/build_sciplatlab.groovy:44]: Something wrong with the indent here.

File: pipelines/sqre/infra/build_sciplatlab.groovy [hunk 3]
```diff
@@ -16,91 +16,57 @@ node('jenkins-manager') {
...
-               ${opts} \
-               '${tag}'
-              docker build .
-          """
-        }
+      def conn = url.openConnection().with { conn ->
+        conn.setRequestMethod('POST')
+        conn.setRequestProperty('Content-Type', 'application/json; charset=utf-8')
+        conn.setRequestProperty('Accept', 'application/vnd.github.v3+json')
+        conn.setRequestProperty('Authorization', "Bearer ${GITHUB_TOKEN}")
```
Comment (ktlim) [inline pipelines/sqre/infra/build_sciplatlab.groovy:57]: Looks correct to me. 👍

File: pipelines/sqre/infra/build_sciplatlab.groovy [hunk 4]
```diff
@@ -16,91 +16,57 @@ node('jenkins-manager') {
...
     }
   } // run
 
-  util.nodeWrap('docker') {
-    timeout(time: timelimit, unit: 'HOURS') {
-      run()
-    }
-  } // util.nodeWrap
+
+  util.withGithubAdminCredentials( {
```
Comment (ktlim) [inline pipelines/sqre/infra/build_sciplatlab.groovy:69]: I think you can leave off the parentheses, syntactically.

**Review by ktlim — APPROVED** [review]
A couple of small tweaks.

File: pipelines/sqre/infra/build_sciplatlab.groovy (hunk 1 above)
Comment (athornton) [reply pipelines/sqre/infra/build_sciplatlab.groovy:50]: derp!

File: pipelines/sqre/infra/build_sciplatlab.groovy (hunk 2 above)
Comment (athornton) [reply pipelines/sqre/infra/build_sciplatlab.groovy:44]: Untabified now....

File: pipelines/sqre/infra/build_sciplatlab.groovy (hunk 4 above)
Comment (athornton) [reply pipelines/sqre/infra/build_sciplatlab.groovy:69]: Let's find out!

**Pull request merged**